content/docs/start/data-management/metrics-parameters-plots.md

```diff
@@ -99,44 +103,36 @@ marked as a `metrics` file with `-M`. Its contents are:
 }
 ```
 
-> DVC doesn't force you to use any specific file names, nor does it enforce a
-> format or structure of a metrics file. It's completely user/case-defined.
-> Refer to `dvc metrics` for more details.
-
-You can view tracked metrics with DVC:
+You can view tracked metrics with `dvc metrics show `:
 
 ```dvc
 $ dvc metrics show
 Path                    avg_prec.test    avg_prec.train    roc_auc.test    roc_auc.train
 eval/live/metrics.json  0.94496          0.97723           0.96191         0.98737
 ```
 
-## Configuring plots
+[metrics file]: /doc/command-reference/metrics#supported-file-formats
+
+## Rendering plots
 
-`evaluate.py` also writes `precision`, `recall`, and `thresholds` arrays
-(obtained using
-[`precision_recall_curve`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_recall_curve.html))
-into the plots files `eval/prc/train.json` and `eval/prc/test.json`:
+`evaluate.py` also writes `precision`, `recall`, and `thresholds` arrays into
+the plots files `eval/prc/train.json` and `eval/prc/test.json`:
 
 ```json
 {
   "prc": [
     { "precision": 0.021473008227975116, "recall": 1.0, "threshold": 0.0 },
-    ...,
+    ...
     { "precision": 1.0, "recall": 0.009345794392523364, "threshold": 0.6 }
   ]
 }
 ```
 
-Similarly, it writes arrays for the
-[roc_curve](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_curve.html)
-and
-[confusion matrix](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html)
-into JSON files in the `eval/live/plots` directory, and an image
-`eval/importance.png` with a feature importance bar chart for additional plots.
+Similarly, it writes arrays for the [`roc_curve`] and [`confusion_matrix`] into
+JSON files in the `eval/live/plots` directory, and an additional
+`eval/importance.png` plot with a feature importance bar chart.
```

Comment:
If we could combine this paragraph with the previous one (before the code block), the section would be more readable: just a p and a code block before we get to the main point (configuring and rendering plots).

- [ ] I'll try ⏳